packages/medusa/src/models/return-reason.ts

```diff
@@ -26,6 +29,20 @@ export class ReturnReason {
   @Column({ nullable: true })
   description: string
 
+  @Column({ nullable: true })
+  parent_return_reason_id: string
+
+  @ManyToOne(() => ReturnReason)
+  @JoinColumn({ name: "parent_return_reason_id" })
+  parent_return_reason: ReturnReason
+
+  @OneToMany(
+    () => ReturnReason,
+    return_reason => return_reason.parent_return_reason,
+    { cascade: ["insert"] }
```

Comment:
Should have cascade `delete` as well, such that if we choose to delete a parent return reason, all children are deleted.